app/build.gradle

```diff
@@ -52,5 +52,4 @@ dependencies {
     compile 'org.honorato.multistatetogglebutton:multistatetogglebutton:0.2.2'
     compile 'com.google.code.gson:gson:2.8.0'
     compile 'junit:junit:4.12'
```

Comment:
fairly certain that this statement is not necessary either. JUnit is only used in test compilation